charts/kubescape-operator/values.yaml

```diff
@@ -605,6 +605,7 @@ nodeAgent:
           fieldPath: spec.nodeName
 
   privileged: false
+  seLinuxType: scp_t
```

Comment:
```suggestion
  seLinuxType: spc_t
```
is that dyslexia?

Reply:
Indeed, I will bring paper from the doctor 😉